src/zoo/owl_zoo_cmd.ml

```diff
@@ -121,12 +135,12 @@ let print_info () =
     "Usage: \n" ^
     "  owl [utop options] [script-file]\texecute an Owl script\n" ^
     "  owl -upload [gist-directory]\t\tupload code snippet to gist\n" ^
-    "  owl -download [gist-id]\t\tdownload code snippet from gist\n" ^
+    "  owl -download [gist-id] [ver-id]\tdownload code snippet from gist; download the latest version if ver-id not specified\n" ^
     "  owl -remove [gist-id]\t\t\tremove a cached gist\n" ^
     "  owl -update [gist-ids]\t\tupdate (all if not specified) gists\n" ^
-    "  owl -run [gist-id]\t\t\trun a self-contained gist\n" ^
-    "  owl -info [gist-ids]\t\t\tshow the basic information of a gist\n" ^
-    "  owl -list\t\t\t\tlist all the cached gists\n" ^
+    "  owl -run [gist-name]\t\t\trun a self-contained gist\n" ^
+    "  owl -info [gist-name]\t\t\tshow the basic information of a gist\n" ^
+    "  owl -list [gist-id]\t\t\tlist all cached versions of a gist; all the cached gists if not specified\n" ^
```

Comment:
What is the difference between `gist-name` and `gist-id`?

Reply:
`gist-name` is the full naming scheme, including `gist-id`, `version-id`, and `pin`.